### ISSUE TITLE
[New feature] show replay of scored goals (after goals ofc)

### ISSUE THREAD

**Issue by Wenomka** [haxball/haxball-issues#133]
Dream for any players who love watching goals scored by himself.

Will be nice if this feature (if it possible for haxball) could be in setting and admins could enable or disable it.

Possible it for html or not - huge question.

**Comment by Leo-MapCreator** [comment]
No, its boring, after every goal you have to wait 5secs of replay, HaxArena have that and its really boring

**Comment by Wenomka** [comment]
i said what it can be disable in settings. Will be good for casters. Its boring for public rooms probably.

**Comment by Kozlov98** [comment]
That is going to slow a simple game, with streaming you can do it with plugins

**Comment by Wenomka** [comment]
@Kozlov98 btw it will be good for fair play after goal if one of players got lag.

**Comment by haxballer** [comment]
If Basro adds a player who will be able to play the goals of the recordings, it will be enough. Do you really think that those who have not scored goals will want to watch the repeat? This will slow down the game. I respect the fact that you have a head full of ideas, but they are a little bit exaggerated.
